src/System.IO.FileSystem/src/System/IO/FileSystem.Windows.cs

```diff
@@ -500,7 +500,7 @@ private static void RemoveDirectoryRecursive(string fullPath, ref Interop.Kernel
                                 if (!Interop.Kernel32.DeleteVolumeMountPoint(mountPoint) && exception == null)
                                 {
                                     errorCode = Marshal.GetLastWin32Error();
-                                    if (errorCode != Interop.Errors.ERROR_SUCCESS && 
+                                    if (errorCode != Interop.Errors.ERROR_SUCCESS &&
                                         errorCode != Interop.Errors.ERROR_PATH_NOT_FOUND)
                                     {
                                         exception = Win32Marshal.GetExceptionForWin32Error(errorCode, fileName);
```

Comment:
Please consider reverting the changes to this file.

Reply:
This is not my file I don't do anything in C# and I don't see any changes in my repos (I do see space change above). It looks like it was part of commit 0e5843dc but the changes were done by opening file for review in VS and I don't even see the change in VS. I will need help undoing a commit.